src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2,4 +2,16 @@ databaseChangeLog:
   - include:
       file: db/changelog/changes/01-create-book-table.yaml
   - include:
-      file: db/changelog/changes/01-create-user-table.yaml
+      file: db/changelog/changes/02-create-users-table.yaml
+  - include:
+      file: db/changelog/changes/03-insert-users.yaml
+  - include:
+      file: db/changelog/changes/04-create-roles-table.yaml
+  - include:
+      file: db/changelog/changes/05-create-users-roles-table.yaml
+  - include:
+      file: db/changelog/changes/06-insert-roles.yaml
+  - include:
+      file: db/changelog/changes/07-assign-roles.yaml
+  - include:
+      file: db/changelog/changes/08-add-is-deleted-column-to-users.yaml
```

Comment:
add empty line at the end of every changelogs

Reply:
May I ask why we need to use RoleRepository if we don't use it anywhere? Is this for the future or do I have a mistake?